treeherder/etl/bugzilla.py

```diff
@@ -36,10 +36,7 @@ def run(self):
         offset = 0
         limit = 500
 
-        # fetch new pages no more than 30 times
-        # this is a safe guard to not generate an infinite loop
-        # in case something went wrong
-        for i in range(1, 30 + 1):
+        while True:
```

Comment:
I must admit that ``while True`` statements usually make me cringe.  But In this case I think it's fine.  The code is simple enough that we can plainly see line 47 and 49 will break out if the request fails, or we hit the end of the list.